src/BitTorrent-1.1.xml

```diff
@@ -0,0 +1,509 @@
+             Modifications. For the avoidance of doubt, to the extent your executable version of a Licensed
+             Product does not contain your or another Contributor&apos;s material Modifications, you may
+             not sell, accept donations or otherwise receive compensation for such executable.</p>
+          <p>You may use the Licensed Product for any purpose, but the Licensor is not providing you any
+             warranty whatsoever, nor is the Licensor accepting any liability in the event that the
+             Licensed Product doesn&apos;t work properly or causes you any injury or damages.</p>
+        </li>
+        <li>
+          <b>6.</b>
+          <p>If you sublicense the Licensed Product or Derivative Works, you may charge fees for warranty or
+             support, or for accepting indemnity or liability obligations to your customers. You cannot
+             charge for, sell, accept donations or otherwise receive compensation for the Source Code.</p>
+        </li>
+        <li>
+          <b>7.</b>
+          <p>If you assert any patent claims against the Licensor relating to the Licensed Product, or if you
+             breach any terms of the License, your rights to the Licensed Product under this License
+             automatically terminate.</p>
+        </li>
+      </list>
+      <p>You may use this License to distribute your own Derivative Works, in which case the provisions of this
+         License will apply to your Derivative Works just as they do to the original Licensed Product.</p>
+      <p>Alternatively, you may distribute your Derivative Works under any other OSI-approved Open Source license,
+         or under a proprietary license of your choice. If you use any license other than this License,
+         however, you must continue to fulfill the requirements of this License (including the provisions
+         relating to publishing the Source Code) for those portions of your Derivative Works that consist of
+         the Licensed Product, including the files containing 
+        <br/>Modifications. 
+      </p>
+      <p>New versions of this License may be published from time to time in connection with new versions of a
+         Licensed Product or otherwise. You may choose to continue to use the license terms in this version of
+         the License for the Licensed Product that was originally licensed hereunder, however, the new versions
+         of this License will at all times apply to new versions of the Licensed Product released by Licensor
+         after the release of the new version of this License. Only the Licensor has the right to change the
+         License terms as they apply to the Licensed Product.</p>
+      <p>This License relies on precise definitions for certain terms. Those terms are defined when they are first
+         used, and the definitions are repeated for your convenience in a Glossary at the end of the
+         License.</p>
+    </optional>
```

Comment:
should we have everything above this as optional?  (we did not have this marked as so before)